<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TOUCH_TEST</class>
 <widget class="QMainWindow" name="TOUCH_TEST">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="touch_test_1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>75</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
   border:none;
   background-color:red;
}
</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="touch_test_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>75</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
   border:none;
   background-color:red;
}
</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="touch_test_3">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>130</y>
      <width>75</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
   border:none;
   background-color:red;
}
</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 15pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="touch_test_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>75</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
   border:none;
   background-color:red;
}
</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="touch_test_5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>310</y>
      <width>75</width>
      <height>65</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton
{
   border:none;
   background-color:red;
}
</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>320</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
